virtual-server/examples/python/kubevirtclient.py

```diff
@@ -0,0 +1,252 @@
+import os
+import six
+
+from kubernetes import client, config, watch
+from kubernetes.client.exceptions import (
+    ApiTypeError,
+    ApiValueError
+)
+
+
+class KubeVirtClient:
```

Comment:
I would love to use kubevirt python library but it's broken for a couple of months, 2 issues are already created but no one is willing to fix them.
This is my short implementation based on k8s lib.